lib/services/keyVault/lib/models/jsonWebKey.js

```diff
@@ -149,7 +150,7 @@ JsonWebKey.prototype.mapper = function () {
         },
         k: {
           required: false,
-          serializedName: 'K',
+          serializedName: 'k',
```

Comment:
It is a bug fix. It wouldn't get deserialize without this change for nodeJs.

---
In reply to: [76497731](https://github.com/Azure/azure-sdk-for-node/pull/1858#discussion_r76497731) [](ancestors = 76497731,76497661,76497637)